### DISCUSSION TITLE
Airtable <> NocoDB features comparison

### DISCUSSION THREAD

**Discussion by Vadorequest** [nocodb/nocodb#196] [Q&A]
Is there any "features comparison overview" anywhere?

As you state yourself to be "the open-source alternative of Airtable", it's important to provide a comprehensive feature comparison for people who want to compare both.

I'd like to know:
- What's possible with Airtable that isn't possible with NocoDB (and if not, whether it's planned to have a feature parity in the future)
  - _This is important for folks (like me) who are considering migrating from Airtable to NocoDB, because those are breaking changes for us and we need to be aware of them._ 
- What's possible with NocoDB that isn't possible with Airtable
  - _This is interesting to know the new features/capabilities I'll gain by using NocoDB!_

Also, it would be great to have a sandbox/demo to "feel" the product ourselves quickly, one of the official Airtable examples/staters would be great to serve as a comparison basis!

**Comment by keskinonur** [comment]
And do we get some basic mathematical, statistical functions?

**Reply by o1lab** [reply]
This is in our immediate roadmap #138

**Reply by o1lab** [reply]
https://github.com/nocodb/nocodb/projects/1
